components/hidden_data_box/commons/hidden_data_box.lua

```diff
@@ -48,17 +52,11 @@ function HiddenDataBox.run(args)
 		if not queryResult[1] and Namespace.isMain() then
 			table.insert(warnings, String.interpolate(INVALID_PARENT, {parent = parent}))
 		elseif args.participantGrabber then
```

Comment:
Is any wiki still using `participantGrabber=false`?

Reply:
broodwar